css-values-4/Overview.bs

```diff
@@ -2499,21 +2499,23 @@ Trigonometric Functions: ''sin()'', ''cos()'', ''tan()'', ''asin()'', ''acos()''
 	and the angle returned by ''atan()'' to the range [''-90deg'', ''90deg''].
 
 	The <dfn lt="atan2()">atan2(A, B)</dfn> function
-	contains two comma-separated [=calculations=] A and B
-	that must resolve to <<number>>s,
-	and returns the <<angle>>
+	contains two comma-separated [=calculations=], A and B.
+	A and B can resolve to any <<number>>, <<dimension>>, or <<percentage>>,
+	but must have the <em>same</em> [=determine the type of a calculation|type=],
+	or else the function is invalid.
+	The function returns the <<angle>>
 	between the positive X-axis and the point (B,A).
 	The returned angle must be normalized to the interval (''-180deg'', ''180deg'']
 	(that is, greater than ''-180deg'', and less than or equal to ''180deg'').
 
 	Note: ''atan2(Y, X)'' is <em>generally</em> equivalent to ''atan(Y / X)'',
-	but it gives a better answer when the point in question is in the second (NW) or third (SW) quadrants of the plane.
-	''atan2(1, -1)'', corresponding to the point (-1, 1) in the NW of the plane,
+	but it gives a better answer when the point in question includes negative components on one dimension.
```

Comment:
Not quite right; fourth quadrant (SE) gives the same results in both functions, despite one negative value.

Reply:
Ok, new wording is "may include negative components". Even if sometimes the value is the same, it is still a "better answer" because you are confident of what it represents.